frontend/wallet/src/render/Theme.re

```diff
@@ -30,8 +30,9 @@ module Colors = {
   let yeezy = `hex("C53131");
 
   let midnight = `hex("1F2D3D");
-  let marineLight = `rgba((51, 104, 151, 0.1));
   let marine = `hex("336897");
+  let marineLight = `rgba((51, 104, 151, 0.1));
```

Comment:
Is there a reason this doesn't use the same "xAlpha" pattern as other colors?

Reply:
Ah yeah that's a good idea.